docs/quickstart/infrastructure.md

```diff
@@ -9,17 +9,22 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/quickstart/
 
 Together, the Netdata Agent and Netdata Cloud create a powerful, infinitely-scalable infrastructure monitoring solution.
 
-The Netdata Agent uses zero-configuration collectors to gather metrics from every application and container instantly.
-The distributed data model lets you monitor everything without a slow and troublesome centralized data lake for your
-infrastructure's metrics, reducing the resources you need to invest in metrics retention. Netdata Cloud unifies all the
-metrics from these distributed nodes on customizable, interactive, and real-time visualizations.
+The Netdata Agent uses zero-configuration collectors to gather metrics from every application and container instantly,
+and uses Netdata's [distributed data architecture](/docs/store/distributed-data-architecture.md) to store metrics
+locally. Without a slow and troublesome centralized data lake for your infrastructure's metrics, you reduce the
+resources you need to invest, and the complexity of, monitoring your infrastructure. 
```

Comment:
invest --> 'invest in' since you're using an preposition for 'complexity of'

Reply:
Thanks much, @aabatangle! Fixed these.